DQMOffline/Configuration/python/DQMOffline_cff.py

```diff
@@ -31,6 +32,9 @@
                                  es_dqm_source_offline *
                                  castorSources *
                                  HcalDQMOfflineSequence )
+eras.phase1Pixel.toReplaceWith(DQMOfflinePreDPG, DQMOfflinePreDPG.copyAndExclude([ # FIXME
+    siPixelOfflineDQM_source, # Pixel DQM needs to be updated for phase1
```

Comment:
@schneiml Taking a look on your branch, I would say that
* this is not a bad place to take out pixel DQM (it could be done also in `SiPixelOfflineDQM_source_cff` but this is more "blind" approach)
* replacing phase0 DQM with phase1 DQM is best done in `SiPixelOfflineDQM_source_cff` as you did

Basically, if/after this PR gets integrated, for enabling the pixel DQM sequence you need to remove these three lines (in addition to the phase0->phase1 replacement you already have).